firmware/application/apps/ui_flash_utility.hpp

```diff
@@ -40,7 +40,7 @@ class FlashUtilityView : public View {
 
     void focus() override;
 
-    std::string title() const override { return "Flash Utility"; };
+    std::string title() const override { return "FlashUtility"; };  // Removed the space because the title and speaker icon overlapped.
```

Comment:
Eh, probably not the best idea considering that the list of icons on the top bar is not constant. Better fix would probably be to truncate the title in the navigation code based on the icons width.

Reply:
Thank you! However I'm not sure about this, Fir example I think "HappyBirthday" is more intuitive than "Happy Birthda", how do you think?

Reply:
*For

Reply:
My point is that on my device, I have the speaker button hidden so I see “Happy Birthday”, you want to shrink it  to “HappyBirthday” for _your_ setup. If you enable options for up/down conversion, that adds another button to the top and now you have “HappyBirthda” again. 

Call it “Flash Util” or something if you want, but it might be better to make the icon bar a fixed width.

Reply:
I'm a minimalist so if it could be done by less code (but with some tricks), ill choose that way. Like my touchable title and this situation. Hahaha forgive me.

Reply:
That’s fine. You can just resolve this one as won’t fix.

Reply:
Thank you! Let's make it a temp implementation.